infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -412,9 +405,29 @@ private void updateLocalMetrics() {
         } else {
             sequencerMetrics = new SequencerMetrics(SequencerStatus.UNKNOWN);
         }
+        return sequencerMetrics;
+    }
+
+    /**
+     * Task to collect local node metrics.
+     * This task collects the following:
+     * Boolean Status of layout, sequencer and logunit servers.
+     * These metrics are then composed into ServerMetrics model and stored locally.
+     */
+    private void updateLocalMetrics() {
+
+        suppressExceptions(() -> {
```

Comment:
i do not think wrapping should be here . It should be at scheduling level. The problem is that scheduler is finicky.